cvat/apps/engine/cloud_provider.py

```diff
@@ -25,29 +26,18 @@
 
 from cvat.apps.engine.log import slogger
 from cvat.apps.engine.models import CredentialsTypeChoice, CloudProviderChoice
+from cvat.apps.engine.utils import DjangoEnum, StrEnum
 
-class Status(str, Enum):
+class Status(DjangoEnum, StrEnum):
```

Comment:
Likewise, both this and `Permissions` aren't used with Django models.

Reply:
Both Status and Permissions have extra methods, that I would like to unify.